lv_conf_template.h

```diff
@@ -695,6 +695,17 @@
     #endif // LV_IME_PINYIN_USE_K9_MODE
 #endif
 
+/*1: Enable Pinyin input method*/
```

Comment:
Why does it require to " Enable Pinyin input method"?

Reply:
modified